CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-05-20
+
+This release is rolling out to all regions - ETA for conclusion 2021-05-27 for public cloud and 2021-05-31 for sovereign clouds.
+
+### Announcements
+
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+
+### Release Notes
+
+* Component Updates
+  * Reverting Container Insights agent to March release [ciprod03262021] in response to failing livenss probes.
```

Comment:
we should probably list this as a bugfix so people find it easy to see the issue and resolution

Reply:
Please add. 

Windows container can now update the timezone within the container. For more detail, please go to the faq. https://docs.microsoft.com/en-us/azure/aks/windows-faq